dist/index.js

```diff
@@ -143,4 +146,4 @@ class MagicConnect extends types_1.Connector {
         });
     }
 }
-exports.MagicConnect = MagicConnect;
```

Comment:
We need to maintain backward compatibility which means we also need to export MagicConnect, not just delete it. Look at the WAGMI connector Josh made as an example. Look at line 110 of the [dedicatedWalletConnector.ts](https://github.com/Unboxed-Software/web3-react-magic-connector/pull/1/files#diff-57cac3d606222f3ea19936c99f49e1719751e3217f78aa4717c495d7651ab175)